core-bundle/src/Resources/contao/templates/backend/be_main.html5

```diff
@@ -117,4 +122,4 @@
   <script>Backend.initScrollOffset()</script>
 
 </body>
-</html>
```

Comment:
You should configure your IDE correctly, this happens in all of your PRs somehow :D